server/routes/restrictions/enter-restriction/enterNewRestrictionController.ts

```diff
@@ -7,8 +7,9 @@ import { Navigation } from '../../contacts/common/navigation'
 import ReferenceCode = contactsApiClientTypes.ReferenceCode
 import RestrictionClass = journeys.RestrictionClass
 import { maxLengthForRestrictionClass, RestrictionSchemaType } from '../schema/restrictionSchema'
+import { formatNameFirstNameFirst } from '../../../utils/formatName'
 
```

Comment:
2 x NameFirst?

Reply:
Could be legit - just reads oddly!

Reply:
Yeah open to suggestions on those method names. It was called reverseNameFormat before but had the first name first... We now have `formatNameFirstNameFirst` and `formatNameLastNameFirst`